README.md

```diff
@@ -84,7 +84,6 @@ Pieman is a core component of [CusDeb](https://cusdeb.com).
   * mkpasswd
   * pandoc
   * rsync
-  * uuidgen
   * User mode emulation binaries such as `/usr/bin/qemu-arm-static` and `/usr/bin/qemu-aarch64-static`
   * wget
```

Comment:
Don't forget to remove it from the list, because there's an alternative to it called `do_wget`.